pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -542,7 +548,8 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 					continue
 				}
 				driverName := stork_api.GenericDriver
-				if backup.Spec.BackupType == stork_api.ApplicationBackupGeneric {
+				if backup.Spec.BackupType == stork_api.ApplicationBackupGeneric ||
+					driverType == stork_api.ApplicationBackupGeneric {
```

Comment:
There is also one place, where we are checking for the genericBackup. 
https://github.com/libopenstorage/stork/blob/master/pkg/applicationmanager/controllers/applicationbackup.go#L590